29 add and update functions for chemical domain endpoints

rachmaninoffpk opened the pull request:
This closes #29 .

brown-jason reviewed (commented):
1. Testing the code from the vignette, chemicals_in_ccl4_exact <- get_chemicals_in_list_exact(list_name = 'CCL4', word = 'Bisphenol A') - Returns an empty list.  Should the example return a populated list?
2. chemical_image_by_smiles returns a large array.  It would be nice to show how to take this array and either save or visualize an image.

rachmaninoffpk commented:
> 1. Testing the code from the vignette, chemicals_in_ccl4_exact <- get_chemicals_in_list_exact(list_name = 'CCL4', word = 'Bisphenol A') - Returns an empty list.  Should the example return a populated list?
> 2. chemical_image_by_smiles returns a large array.  It would be nice to show how to take this array and either save or visualize an image.

I just adjusted the `get_chemicals_in_list_exact()` example to output a populated list and added some example code for displaying an image from the large array returned by `chemical_image_by_smiles()`.
